packages/e2e-tests/src/tests/WrappedKeys.spec.ts

```diff
@@ -0,0 +1,1211 @@
+
+    const { pkpAddress, id } = await importPrivateKey({
+      pkpSessionSigs: pkpSessionSigs!,
+      privateKey: privateKey1,
+      litNodeClient: devEnv.litNodeClient!,
+      publicKey: '0xdeadbeef',
+      keyType: 'K256',
+      memo: 'Test Key',
+    });
+
+    const alicePkpAddress = alice.authMethodOwnedPkp?.ethAddress;
+    expect(pkpAddress).toEqual(alicePkpAddress);
+    try {
+      const privateKey2 = randomSolanaPrivateKey();
+
+      await importPrivateKey({
+        pkpSessionSigs: pkpSessionSigs!,
+        privateKey: privateKey2,
+        litNodeClient: devEnv.litNodeClient!,
+        publicKey: '0xdeadbeef',
+        keyType: 'K256',
+        memo: 'Test Key',
+      });
+    } catch (e: any) {
+      if (
+        e.message.includes(
+          'There is already a wrapped key stored, either for the provided pkpAddress, or with the same dataToEncryptHash; a pkpAddress may only have 1 wrapped key, and a wrapped key may only be associated with a single pkpAddress.'
+        )
+      ) {
+        console.log('✅ THIS IS EXPECTED: ', e);
+      } else {
+        throw e;
+      }
+    }
+  });
+
+  it('Fail Max Expiration Session Sig', async () => {
+    const alice = await devEnv.createRandomPerson();
+
+    const pkpSessionSigs = await getPkpSessionSigs(devEnv, alice);
+
+    console.log(pkpSessionSigs);
+
+    try {
+      const privateKey = randomSolanaPrivateKey();
+
+      await importPrivateKey({
+        pkpSessionSigs: pkpSessionSigs!,
+        privateKey,
+        litNodeClient: devEnv.litNodeClient!,
+        publicKey: '0xdeadbeef',
+        keyType: 'K256',
+        memo: 'Test Key',
+      });
+    } catch (e: any) {
+      if (e.message.includes('Expires too far in the future')) {
+      } else {
+        throw e;
+      }
+    }
+  });
+
+  it('Fail Import Invalid Session Sig', async () => {
+    const tamperPkpSessionSigs = (
+      pkpSessionSig: SessionSigsMap
+    ): SessionSigsMap => {
+      const tamperedPkpSessionSigs: SessionSigsMap = {};
+
+      for (const key in pkpSessionSig) {
+        if (pkpSessionSig.hasOwnProperty(key)) {
+          const authSig = pkpSessionSig[key];
+          const updatedAuthSig: AuthSig = {
+            ...authSig,
+            address: authSig.address.slice(0, -1),
+          };
+          tamperedPkpSessionSigs[key] = updatedAuthSig;
+        }
+      }
+
+      return tamperedPkpSessionSigs;
+    };
+
+    const alice = await devEnv.createRandomPerson();
+
+    const pkpSessionSigs = await getPkpSessionSigs(devEnv, alice);
+
+    try {
+      const privateKey = randomSolanaPrivateKey();
+
+      await importPrivateKey({
+        pkpSessionSigs: tamperPkpSessionSigs(pkpSessionSigs!),
+        privateKey,
+        litNodeClient: devEnv.litNodeClient!,
+        publicKey: '0xdeadbeef',
+        keyType: 'K256',
+        memo: 'Test Key',
+      });
+    } catch (e: any) {
+      if (e.message.includes('bad public key size')) {
```

Comment:
We can use expect.toThrow() here instead, and :hocho: the extra `console.log`
See also https://jestjs.io/docs/expect#rejects :)